src/Console/stubs/field_simple.stub

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace DummyNamespace;
+
+class DummyClass
+{
+    /**
+     * @param  null  $_ Always `null`.
```

Comment:
Shouldn't this comment be the same than in field_full.stub?
```
 @param  null  $rootValue Usually contains the result returned from the parent field. In this case, it is always `null`.
```

Reply:
We might shorten both to:

```suggestion
     * @param  null  $_
```

Reply:
I agree

Reply:
I decided to strip all descriptions from the simple version and include them in the full version.